src/pages/NotificationsPage/components/NotificationItem.tsx

```
@@ -0,0 +1,120 @@
+          </span>
+        </div>
+
+        <p className="text-sm leading-relaxed text-gray-600">
+          {message}
+          {actionLabel && (
+            <>
+              {" "}
+              <button
+                onClick={onAction}
+                className="font-semibold text-primary hover:underline"
+              >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)